documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
+
+## Performance Optimisation
+
+### Enable Asynchronous Querying
+To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the UI to remain responsive. You can deploy additional Superset workers to offload and process background queries. Deploy Charmed Superset workers to the same model as the server with:
```

Comment:
```suggestion
To enable asynchronous querying, you need to deploy Charmed Superset Workers. These workers handle long-running queries asynchronously, allowing the User Interface (UI) to remain responsive. You can deploy additional workers to offload and process background queries. 

Deploy Charmed Superset workers to the same model as the server as follows:
```

Reply:
Can we add a hyperlink to [Charmed Superset Workers](link) for further info to the user?

Reply:
In general, define acronyms first time they appear

Reply:
No, we don't have a page dedicated to the workers at this time. The worker is just a configuration change for the superset-k8s charm.